src/account/PluginManager2.sol

```diff
@@ -101,7 +101,7 @@ abstract contract PluginManager2 {
         // TODO: consider enforcing this from user-supplied install config.
         for (uint256 i = 0; i < selectors.length; ++i) {
             bytes4 selector = selectors[i];
-            if (!_storage.selectorData[selector].validations.remove(toSetValue(validationFunction))) {
+            if (!_storage.validationData[validationFunction].selectors.remove(toSetValue(selector))) {
```

Comment:
hmm can we delete the entire set without passing it in from calldata?

Reply:
Yes, we can now. Updated. This is a nice surprise from the change, and makes it simpler for us.